vue/components/organisms/pickers/pickers.vue

```diff
@@ -438,7 +438,13 @@ export const Pickers = {
             /**
              * Whether the colors picker has a scroll.
              */
-            scrollColor: false
+            scrollColor: false,
+            /**
+             * The currently centered part, material and
+             * color, necessary to maintain the scroll
+             * after a resize.
+             */
+            scrollCenterElements: {}

```

Comment:
Make this more generic so that it can cope with different strategies (such as aligning right). Something like `alignedElements`.